src/data/script.ts

```diff
@@ -58,14 +58,30 @@ export interface WaitAction {
   timeout?: number;
 }
 
+export interface RepeatAction {
+  repeat: {
+    count?: number;
+    while?: Condition[];
+    until?: Condition[];
```

Comment:
We shouldn't combine them. Instead make 3 interfaces and a type that is an "or". 

```ts
export type RepeatAction = CountRepeatAction | WhileRepeatAction | UntilRepeatAction
```